tests/pools/abstract/abstract-pool.test.js

```diff
@@ -61,4 +61,40 @@ describe('Abstract pool test suite', () => {
       const pool = new StubPoolWithIsMainMethod(1, '').toThrowError()
     })
   })
+
+  it('Verify that numberOfWorkers is checked', () => {
+    expect(() => {
+      const pool = new FixedThreadPool(
+        './tests/worker-files/thread/testWorker.js'
+      ).toThrowError(
```

Comment:
And this may be the error here :warning: 
It should be called on expected and not inside it! The tests are running without doing anything!